public/pages/AnomalyCharts/containers/AnomalyHeatmapChart.tsx

```diff
@@ -61,6 +61,7 @@ interface AnomalyHeatmapChartProps {
   onDisplayOptionChanged?(option: HeatmapDisplayOption | undefined): void;
   heatmapDisplayOption?: HeatmapDisplayOption;
   entityAnomalySummaries?: EntityAnomalySummaries[];
+  isNotSample: boolean;
```

Comment:
You may make this optional for any backwards compatibility issues.

Reply:
sure.